run-nextflow-tests/utils.py

```diff
@@ -170,6 +173,17 @@ def assign_value(closure, key, value):
 
             assign_value(closure[local_key], remainder, value)
 
+    # Parse out the current manifest version
+    try:
+        version_str = re.search(
+            r"^manifest.version=(.*)$",
+            config_str,
+            re.MULTILINE
```

Comment:
Is the flag here to force it match the pattern on a single line and avoid potential issues with `(.*)$` matching all the way to the end of the string across multiple lines?

Reply:
By default `^` matches the very beginning of the string and `$` matches the very end. `re.MULTILINE` makes them match just after/before newlines as well.

I am not using `re.DOTALL`, which would allow `.` to match newlines, so by default the pattern can only match a single complete line.